src/mount/mount-tool.c

```diff
@@ -477,7 +477,7 @@ static int parse_argv(int argc, char *argv[]) {
                                 }
 
                                 if (arg_transport == BUS_TRANSPORT_LOCAL && arg_canonicalize) {
-                                        r = chase(p, /* root= */ NULL, /* flags= */ 0, &arg_mount_what, /* ret_fd= */ NULL);
+                                        r = chase(p, /* root= */ NULL, /* chase_flags= */ 0, &arg_mount_what, /* ret_fd= */ NULL);
```

Comment:
hmm, i prefer the other way around, i.e rename this to `flags`. there's only one flag field after all, and it brings chase() and chaseat() in line.

Reply:
Looking at `chase.h` it looks like `chaseat` is the odd one out as the rest all use `chase_flags` so wouldn't it be more logical to rename `chaseat`?

Reply:
I'd like to make chase() and chaseat() name `flags` rather than `chase_flags`, as it is trivial that the flags are `ChaseFlag`. But for `chase_and_xyz()` and friends, some of them takes other flags, e.g. open_flags, hence I'd like to name the argument `chase_flags`, even only one flags argument is taken like chase_and_stat().

Reply:
Thanks for the detailed explanation, pushed a new version.